jscomp/stdlib/camlinternalFormat.js

```diff
@@ -673,7 +673,7 @@ function string_of_formatting_lit(formatting_lit) {
       case 1 : 
           return formatting_lit[0];
       case 2 : 
-          return "@" + $$String.make(1, formatting_lit[0]);
+          return "@" + Caml_string.bytes_to_string(Bytes.make(1, formatting_lit[0]));
```

Comment:
`Caml_string.bytes_to_string @@ Bytes.make (1, ..)`

can be optimized as (String.fromCharCode ...